File: src/1672._Richest_Customer_Wealth.java
https://leetcode.com/problems/richest-customer-wealth/
// Algo 1 : Iterate over matrix : O(n) , O(1)
// Iterate over matrix and keep track of maxWealth for each row.
class Solution {
    public int maximumWealth(int[][] accounts) {
        int m = accounts.length , n = accounts[0].length;
        int maxWealth = 0 , currentCustomerWealth = 0;
        for(int i = 0 ; i < m ; i++) {
            currentCustomerWealth = 0;
            for(int j = 0 ; j < n ; j++) {
                currentCustomerWealth += accounts[i][j];
            }
            if(maxWealth < currentCustomerWealth) {
                maxWealth = currentCustomerWealth;
            }
        }
        return maxWealth;
    }
}

File: src/python/1672._Richest_Customer_Wealth.py
#https://leetcode.com/problems/richest-customer-wealth/

from typing import List

# Algo 1 : Iterate over matrix : O(mn) , O(1)
# Iterate over matrix and keep track of maxWealth for each row.
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        richest = 0
        for account in accounts:
            money_in_account = 0
            for money in account:
                money_in_account += money
            richest = max(richest, money_in_account)
        return richest
                